src/main/kotlin/carracing/App.kt

```diff
@@ -1,13 +1,15 @@
 package carracing
 
 fun main() {
-    println("How many cars are in the race?")
-    val numCars = readlnOrNull()?.toIntOrNull() ?: 3
+    println("Enter the names of the cars (separated by commas):")
+    val carNamesInput = readlnOrNull().orEmpty()
+    val carNames = carNamesInput.split(",").map { it.trim() }
 
     println("How many rounds will be played?")
     val numRounds = readlnOrNull()?.toIntOrNull() ?: 5
 
     println("Race Results:")
-    val race = Race(numCars, numRounds)
+    val race = Race(carNames, numRounds)
     race.start()
+    race.displayWinners()
```

Comment:
What do you think about introducing a Winner role or object?
Do you see any potential benefits in doing so?